plugins/package-managers/pub/src/funTest/assets/projects/synthetic/pub-expected-output-multi-module.yml

```diff
@@ -1,6 +1,6 @@
 ---
 projects:
-- id: "Pub:multimodule:multimodule:1.0.0"
+- id: "Pub::multimodule:1.0.0"
```

Comment:
Commit message: At first I though "The feature has been requested several times" refers to a request towards ORT to remove Pub namespaces. Maybe just put the link references after "Pub does not support namespaces for packages" and omit the second sentence completely?